src/definition/metadata/AppMethod.ts

```diff
@@ -43,4 +43,6 @@ export enum AppMethod {
     EXECUTEPREROOMDELETEPREVENT = 'executePreRoomDeletePrevent',
     CHECKPOSTROOMDELETED = 'checkPostRoomDeleted',
     EXECUTEPOSTROOMDELETED = 'executePostRoomDeleted',
+    // Blockit handlers
+    BLOCKIT_ACTION = 'executeActionHandler',
```

Comment:
Is `executeActionHandler` a typo? Can you please consider using `executeBlockitActionHandler`?

Reply:
This is more of a Proof of Concept, the `blockit` portion is probably not going to stick. Most namings might get a revamp before launch ;)